response_operations_ui/templates/conversation-view.html

```diff
@@ -97,11 +97,22 @@ <h1 class="saturn" name="page-messages-title">{{ subject }}</h1>
                                  maxlength='10000',
                                  cols=80) }}
                 </div>
-                <br />
-                <input class="btn" id="btn-send-message" name="send" type="submit" value="Send" />
-                <br />
+                <input class="btn u-mt-s" id="btn-send-message" name="send" type="submit" value="Send" />
+                <a href="{{ url_for('messages_bp.close_conversation', thread_id=messages[0].thread_id) }}"
+                   class="btn btn--secondary btn--border u-mt-s u-ml-xs" id="btn-close-conversation" >Close conversation</a>
             </form>
-          <a href="{{ url_for('messages_bp.view_selected_survey', selected_survey=selected_survey, page=page) }}">< Back</a>
         </div>
+        {% else %}
+        <div class="panel panel--simple panel--info">
+            <div class="panel__body">
+                <p class="mars">This conversation was closed on {{ thread_data.closed_at }} by {{ thread_data.closed_by }}</p>
+            </div>
+        </div>
+        <form action="{{ url_for('messages_bp.close_conversation', thread_id=messages[0].thread_id, selected_survey=selected_survey, reopen_conversation=True) }}" method="post">
```

Comment:
This form action should be on the view conversation route, rather than the close conversation route, as that is where the action takes place.

Also, selected_survey shouldn't be needed as it's handled by the conversation listing view.